exporters/BxDRobotExporter/robot_exporter/JointResolver/ControlGUI/SynthesisGUI.cs

```diff
@@ -66,7 +66,7 @@ public static RuntimeMeta CreateRuntimeMeta()
     public RigidNode_Base SkeletonBase = null;
     public List<BXDAMesh> Meshes = null;
     public bool MeshesAreColored = false;
-    public float TotalMass = 120;
+    public float TotalWeightKg = -1; // Negative value indicates default mass should be left alone
```

Comment:
Generally public variables like this are made properties (the C# equivalent of adding a getter and setter). Not a huge issue but it would be nice to follow C# conventions.